docs/versioning/how-to-version.md

```diff
@@ -0,0 +1,85 @@
+| :----------------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-registry-create-reason.png" alt="drawing" width="600"/> |
+
+Upon clicking the "Submit" button, the new version of the item will be created. 
+A set of jobs will be sent to be processed in order for the provenance graphs to be created and updated.
+
+The page will be refreshed to the version of the item you just created.
+
+## Step 5. View the version info
+
+You now have a new version of the item. You can view the version history including the current and previous versions.
+
+|               Updated versioning details                                                    |
+| :----------------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-registry-create-version-result.png" alt="drawing" width="600"/> |
+
+You can click on the icon with 3 dots in the versioning timeline to expand to display additional versions.
+
+<img src="../assets/images/versioning/versioning-registry-view-expand.png" alt="drawing" width="400"/>
+
+Sorting is available in the drop down menu.
+
+<img src="../assets/images/versioning/versioning-registry-view-sort.png" alt="drawing" width="400"/>
+
+Clicking on the link of the ID of the version will enable you to navigate to that particular version.
+
+## Step 6. Explore version lineage
+
+
+If you would like to view the lineage graph of the item, click on the "EXPLORE LINEAGE" button to view the lineage.
+
+|             Select Explore Lineage button                                     |
+| :----------------------------------------------------------------------------------------------------: |
+|<img src="../assets/images/versioning/versioning-registry-view-lineage.png" alt="drawing" width="600"/> |
+
+You will see the direct lineage of the item in the provenance graph viewer as shown below.
+
+|              Explore Lineage in Provenance Store                                     |
+| :----------------------------------------------------------------------------------------------------: |
+|<img src="../assets/images/versioning/versioning-registry-explore-lineage.png" alt="drawing" width="600"/> |
+
+
+To view previous versions, you can use your mouse to double-click on the item node to expand other related nodes. 
+
+If you continue expanding the nodes including the version activity node, and respective items, you will be able to trace its version lineage in the provenance graph viewer as shown below.
+
+|             Explore version provenance lineage graph                                     |
+| :----------------------------------------------------------------------------------------------------: |
+|<img src="../assets/images/versioning/prov-store-version-lineage.gif" alt="drawing" width="600"/> |
```

Comment:
Cool gif - can we make it bigger

Reply:
i guess we could remove the fixed width attribute?

Reply:
removed